server.go

```diff
@@ -228,12 +228,14 @@ func (u *Upgrader) Upgrade(w http.ResponseWriter, r *http.Request, responseHeade
 	return c, nil
 }
 
-// Upgrade upgrades the HTTP server connection to the WebSocket protocol.
+// DEPRECATED - use websocket.Upgrader instead.
```

Comment:
It would be good to change this line to  `Deprecated: use websocket.Upgrader instead.` in order to conform to the official godoc recommendations.